srcpkgs/caddy/template

```diff
@@ -1,28 +1,32 @@
 # Template file for 'caddy'
 pkgname=caddy
-version=1.0.5
+version=2.3.0
 revision=1
 build_style=go
-# XXX: use caddy/v2 when updating
-go_import_path=github.com/caddyserver/caddy
-go_package="${go_import_path}/caddy"
-hostmakedepends="git"
+go_import_path=github.com/caddyserver/caddy/v2
+go_package="${go_import_path}/cmd/caddy"
 short_desc="Fast, cross-platform HTTP/2 web server with automatic HTTPS"
 maintainer="Dominic Monroe <monroef4@googlemail.com>"
 license="Apache-2.0"
 homepage="https://caddyserver.com"
 distfiles="https://github.com/caddyserver/caddy/archive/v${version}.tar.gz"
-checksum=0e7dc07e4f61f9a00a4c962755098e19ebf8c8a8e0d72e311597ce021b7a2a5e
+checksum=4688b122ac05be39622aa81324d1635f1642e4a66d731e82d210aef78cf2766a
 
 system_accounts="caddy"
+caddy_homedir="/var/lib/caddy"
+caddy_descr="caddy daemon"
+
+conf_files="/etc/caddy/Caddyfile"
+
 make_dirs="
 	/etc/caddy 0700 caddy caddy
 	/var/lib/caddy 0700 caddy caddy"
 
-pre_build() {
-	vsed -e 's,var EnableTelemetry = true,var EnableTelemetry = false,g' -i caddy/caddymain/run.go
+post_extract() {
+	vsed -e "s,unknown,${version},g" -i ${wrksrc}/caddy.go
 }
 
 post_install() {
 	vsv caddy
+	vinstall ${FILESDIR}/Caddyfile 600 etc/caddy
```

Comment:
I think carrying an empty file doesn't make sense here. You can use `vmkdir`, `touch` and `chmod` to create the file and directory without having to install it.

Reply:
Actually the file isn't allowed to be completely empty. 
It needs an empty line, otherwise caddy fails (strange behavior if you ask me, but yeah).